docs/flow/monitoring/debugging.md

```diff
@@ -6,3 +6,58 @@ weight: 300
 ---
 
 # Debugging
+
+There are two primary ways to debug issues with Grafana Agent Flow:
+
+1. Through the Grafana Agent Flow UI
+2. Through examining logs
+
+## Grafana Agent Flow UI
+
+Grafana Agent Flow includes an embedded UI viewable from Grafana Agent's HTTP
+server, which defaults to listening at `http://localhost:12345`.
+
+> The documentation for the [`agent run`][agent run] command describes how to
+> modify the address Grafana Agent listens on for HTTP traffic.
+
+This UI provides two views:
+
+* The home page shows a table of running components along with their health.
+  Clicking the "View" button navigates to the detail page for that component.
+* The "Graph" page shows a graph view of all running components along with
+  their health. Clicking a component in the graph navigates to the detail page
+  for that component.
```

Comment:
UI elements should be referenced in bold rather than quotation marks: https://developers.google.com/style/ui-elements#formatting

(cc @karengermond for a heads up, I stumbled on this while reading the style guide)